source/class/qx/tool/cli/Cli.js

```diff
@@ -34,6 +33,8 @@ qx.Class.define("qx.tool.cli.Cli", {
       throw new Error("qx.tool.cli.Cli has already been initialized!");
     }
     qx.tool.cli.Cli.__instance = this;
+    // use node console log appender with colors
+    qx.log.appender.NodeConsole.setUseColors(true);
```

Comment:
This looks to be hard-coding that colors are to be used with any instance of Cli. It should be using a value obtained from the user. Defaulting to `true` is fine as long as the user has control to turn it off.

Reply:
You are right. I will add a CLI argument to turn it on explicitly.

Reply:
Done.